### ISSUE TITLE
The BC5CDR dataset result

### ISSUE THREAD

**Issue by ZhangYi0621** [zjunlp/DocuNet#18]
Hello!
When I run your code with cdr dataset,get the result like this
![image](https://user-images.githubusercontent.com/67193167/164252838-36b26f68-be3d-4ab3-80b4-049334dee54a.png)
Do you know why it's so high compared to your paper result and previous works!

Looking forward for your reply!
Thanks~

**Comment by ZhangYi0621** [comment]
> Hi, after the paper submission, we tune the hyper-parameters (more training steps, different learning rate, larger batch sizes, and so on) and also obtain better results on BC5CDR; so it is ok to obtain better results.

OK, thank you very much!
